### PR TITLE
Sturgis

### PR DESCRIPTION
**Pull request by Bird2010**
Please review this @casey-collab

[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/2152)

### CODE REVIEW

File: english/sturgis2015/sturgis.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+You jump on your motorcycle and head out for Sturgis, SD!
```
Comment (casey-collab) [inline english/sturgis2015/sturgis.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

**Comment by casey-collab** [comment]
It looks like the file `english/sturgis2015/SturgisPhoto.jpg` is empty.  Please make sure you don't add empty files to the repository.  You could add one sentence to this file, or delete the option that leads to it.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by Bird2010** [comment]
Please review this @casey-collab

**Comment by casey-collab** [comment]
Looks great! :sunny:

**Pull request merged**